### PR TITLE
Update pull_request.yaml after KU-1195

### PR DESCRIPTION
**Pull request by maci3jka**

### CODE REVIEW

**Review by bschimke95 — COMMENTED** [review]
Please rerun with workflow fix

File: .github/workflows/pull_request.yaml [hunk 1]
```diff
@@ -39,6 +39,7 @@ jobs:
     name: Combine Rocks and Push Multiarch Manifest
     uses: canonical/k8s-workflows/.github/workflows/assemble_multiarch_image.yaml@main
     needs: [build-and-push-arch-specifics, run-tests, scan-images]
+    if: ${{ needs.build-and-push-arch-specifics.outputs.changed-rock-metas != '[]' }}
     with:
-      rock-metas: ${{ needs.build-and-push-arch-specifics.outputs.rock-metas }}
+      rock-metas: ${{ needs.build-and-push-arch-specifics.outputs.changed-rock-metas }}
```
Comment (maci3jka) [inline .github/workflows/pull_request.yaml:44]:
```suggestion
      rock-metas: ${{ needs.build-and-push-arch-specifics.outputs.changed-rock-metas }}
      
```

**Review by bschimke95 — APPROVED** [review]
LGTM

File: .github/workflows/pull_request.yaml [hunk 2]
```diff
@@ -39,6 +39,8 @@ jobs:
...
+
```
Comment (maci3jka) [inline .github/workflows/pull_request.yaml]:
```suggestion
```

**Pull request merged**